go-controller/pkg/factory/factory.go

```diff
@@ -70,16 +70,30 @@ const (
 	defaultNumEventQueues uint32 = 15
 )
 
+// types for dynamic handlers created when adding a network policy
+type peerServiceType struct{}
```

Comment:
what about just dropping the "Type" suffix here, then you dont have to add the "Var" below. Then your vars will have the consistent naming scheme as the other types like:

```
	EgressFirewallType                    reflect.Type = reflect.TypeOf(&egressfirewallapi.EgressFirewall{})
	EgressIPType                          reflect.Type = reflect.TypeOf(&egressipapi.EgressIP{})
	CloudPrivateIPConfigType              reflect.Type = reflect.TypeOf(&ocpcloudnetworkapi.CloudPrivateIPConfig{})
	PeerServiceType                    reflect.Type = reflect.TypeOf(&peerService{})
	PeerNamespaceAndPodSelectorType    reflect.Type = reflect.TypeOf(&peerNamespaceAndPodSelector{})
```

Reply:
Sure! It definitely looks better this way.